windows_sg.tf

```diff
@@ -74,3 +74,17 @@ resource "aws_security_group_rule" "windows_ingress_from_kali_instances" {
   from_port                = 0
   to_port                  = 65535
 }
+
+# Allow egress from Windows to Teamserver instances on ports 5000-5999
+# (TCP only).  This port range was requested for use by assessment
+# operators in cisagov/cool-system-internal#127.
```

Comment:
This aligns with the comment describing the `aws_security_group_rule.teamserver_ingress_from_windows_instances_via_5000_to_5999_tcp` resource.

```suggestion
# Allow egress from Windows instances to Teamserver instances on
# ports 5000-5999 (TCP only).  This port range was requested for use
# by assessment operators in cisagov/cool-system-internal#127.
```